<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(194, 237, 206);</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>780</width>
     <height>381</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>765</width>
      <height>379</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="order_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ORDERS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reload_btn">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>20</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#reload_btn{
background-color:white
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>reload_icon.png</normaloff>reload_icon.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_order_line_edit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>19</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#search_order_line_edit{
background-color:white;
padding-left:10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Search With Order status</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
